packages/dnb-eufemia/src/shared/colors.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * DNB UI / Sbanken / DNB Eiendom colors
+ */
+
+export type ColorEntry = {
+    name: string;
+    cssVariable: string;
+    hex: string;
+    rgb: string;
+  };
+  
+  function createColorEntry(name: string, hex: string, rgb: string, cssVariable: string): ColorEntry {
+    return { name, hex, rgb, cssVariable };
+  }
+  
+  export type UiColor = {
+    oceanGreen: ColorEntry;
+    emeraldGreen: ColorEntry;
+    seaGreen: ColorEntry;
+    mintGreen: ColorEntry;
+    summerGreen: ColorEntry;
+    accentYellow: ColorEntry;
+    indigo: ColorEntry;
+    violet: ColorEntry;
+    skyBlue: ColorEntry;
+    lavender: ColorEntry;
+    sandYellow: ColorEntry;
+    pistachio: ColorEntry;
+    seaGreen30: ColorEntry;
+    mintGreen50: ColorEntry;
+    mintGreen25: ColorEntry;
+    mintGreen12: ColorEntry;
+    accentYellow30: ColorEntry;
+    signalOrange: ColorEntry;
+    fireRed: ColorEntry;
+    fireRed8: ColorEntry;
+    successGreen: ColorEntry;
+    black: ColorEntry;
+    black80: ColorEntry;
+    black55: ColorEntry;
+    black20: ColorEntry;
+    black8: ColorEntry;
+    black3: ColorEntry;
+    white: ColorEntry;
+  };
+  
+  export type SbankenColor = {
+    profilePurple: ColorEntry;
+    profilePurpleAlternative: ColorEntry;
+    profileGreen: ColorEntry;
+    profileRed: ColorEntry;
+    profileMagenta: ColorEntry;
+    profileOrange: ColorEntry;
+    profileYellowDark: ColorEntry;
+    profileYellow: ColorEntry;
+    profileGreenDark3: ColorEntry;
+    profileGreenDark2: ColorEntry;
+    profileViolet: ColorEntry;
+    profileVioletLight: ColorEntry;
+    profileBlueDark2: ColorEntry;
+    profileBlueDark: ColorEntry;
+    profileBlue: ColorEntry;
+    uxBlack: ColorEntry;
+    uxText: ColorEntry;
+    uxGrayDark3: ColorEntry;
+    uxGrayDark2: ColorEntry;
+    uxGrayDark: ColorEntry;
+    uxGray: ColorEntry;
+    uxGrayLight: ColorEntry;
+    uxGrayLight2: ColorEntry;
+    uxGrayLight3: ColorEntry;
+    uxGrayDark3Neutral: ColorEntry;
+    uxGrayDark2Neutral: ColorEntry;
+    uxGrayDarkNeutral: ColorEntry;
+    uxGrayNeutral: ColorEntry;
+  };
+  
+  export type EiendomColor = {
+    emeraldGreen50: ColorEntry;
+    emeraldGreen25: ColorEntry;
+    emeraldGreen10: ColorEntry;
+    mintGreen12: ColorEntry;
+  };
+  
+  export const ui: UiColor = {
+    oceanGreen: createColorEntry('Ocean green', '#00343E', '0 52 62', '--color-ocean-green'),
+    emeraldGreen: createColorEntry('Emerald green', '#14555A', '20 85 90', '--color-emerald-green'),
```

Comment:
```suggestion
  emeraldGreen50: {
    name: 'Emerald green 50%',
    hex: '#89aaac',
    rgb: '137 170 172',
    cssVariable: '--color-emerald-green-50',
  },```

Could also do this instead.

Reply:
Could also do this instead.